CRM/Event/DAO/Event.php

```diff
@@ -718,13 +725,15 @@ public static function &fields() {
         ],
         'event_start_date' => [
           'name' => 'start_date',
-          'type' => CRM_Utils_Type::T_DATE + CRM_Utils_Type::T_TIME,
+          'type' => CRM_Utils_Type::T_TIMESTAMP,
           'title' => ts('Event Start Date'),
           'description' => ts('Date and time that event starts.'),
+          'required' => FALSE,
           'import' => TRUE,
           'where' => 'civicrm_event.start_date',
           'headerPattern' => '/^start|(s(tart\s)?date)$/i',
           'export' => TRUE,
+          'default' => 'NULL',
```

Comment:
How are you merging my other PR in? To do that I would only take the change in dao.tpl, and then re-run gencode.php. Then this line should automatically come out as NULL instead of 'NULL', and you'll get the other changes too.

Reply:
Cherry picked it on top, but didn't think to run GenCode.php
My bad, a bit rushed at the moment.